app/src/main/java/com/sdp/movemeet/Sport.java

```diff
@@ -5,5 +5,6 @@
     Soccer,
     Swimming,
     Tennis,
-    Badminton
+    Badminton,
+    Windsurfing
```

Comment:
Seems like I was not the only one to have to add the "Windsurfing" activity ^^'

Reply:
Oh! I wonder why only that one was required, considering many were added at once.